INSTALLING.md

```diff
@@ -151,7 +153,27 @@ model: llama-guard-3-1b (meta-llama/Llama-Guard-3-1B)
 model: llama-guard-3-8b (meta-llama/Llama-Guard-3-8B)
 ```
 
-### 4. Install the Application
+### 4. Configure API Keys
+
+To enable enhanced web search capabilities, you can configure your Tavily API key:
+
+1. **Get a Tavily API Key**: Sign up at [Tavily](https://tavily.com/) and obtain your API key
+
+2. **During installation**: The installer will prompt you to enter your Tavily API key
+   - Enter your key when prompted to enable web search capabilities
+   - Press Enter (leave blank) to skip and proceed without web search functionality
+
+3. **Alternative - Set as environment variable** (for automation/scripting):
+   ```bash
+   export TAVILY_API_KEY="your-tavily-api-key"
+   ```
+   This will skip the interactive prompt during installation.
+
+**Notes**:
+- The application will function without the Tavily key, but web search capabilities will be disabled
+- Setting the environment variable locally only affects the installation process - the key gets deployed to your cluster during installation
```

Comment:
may be better to update `.env.example file.` with this configuration.

Reply:
Done